lambdaclass/mirra_backend
# [GH-463] fixes to levels with costs

## Motivation

Part of https://github.com/lambdaclass/afk_gacha_game/issues/463

## Summary of changes

- Preload the attempt_cost currency of a level before sending the campaign to the client.
- Fix the index of the slot of the dungeon levels units.
- Add max_units of a level to the proto message

## How to test it?

Start the client with this branch: https://github.com/lambdaclass/afk_gacha_game/pull/477
You should be able to spend up to 5 Supplies playing dungeon levels, and see the cost of each level in the line up scene.
You shouldn't be able to play a dungeon level if you exceed the units limit.

## Checklist
